extension/src/server/GradleServer.ts

```diff
@@ -80,6 +94,11 @@ export class GradleServer {
         return this.ready;
     }
 
+    public async getBundleDirectory(): Promise<string> {
+        const extensionPath = await vscode.commands.executeCommand<string>(GET_EXTENSION_PATH);
```

Comment:
At line 33, the extension context is already a member of this class, there is no need to setup a command for this purpose